Pod/Classes/Playback/public/WistiaFlatPlayerView.swift

```diff
@@ -27,5 +28,19 @@ public class WistiaFlatPlayerView: UIView {
             (self.layer as! AVPlayerLayer).player = wistiaPlayer?.avPlayer
         }
     }
+    
+    public func prepareForBackgroundPlayback() {
+        if let playerLayer = layer as? AVPlayerLayer {
```

Comment:
Since this class assumes and relies upon the layer being an `AVPlayerLayer`, I prefer casting `as! AVPlayerLayer` and exploding if something isn't right.  These methods shouldn't fail quietly; at least not in dev.  

Another option would be to add a precondition/assert before the `if let` if you favor that approach.